src/custom/components/Tokens/TokensTableRow.tsx

```diff
@@ -164,22 +166,24 @@ const DataRow = ({
           <ResponsiveLogo currency={tokenData} />
         </RowFixed>
 
-        <TokenText>
-          <LargeOnly style={{ marginLeft: '10px' }}>
-            <Label>{tokenData.symbol}</Label>
-          </LargeOnly>
-
-          <HideLarge style={{ marginLeft: '10px' }}>
-            <RowFixed>
-              <Label fontWeight={400} ml="8px" color={theme.text1}>
-                {tokenData.name}
-              </Label>
-              <Label ml="8px" color={theme.primary5}>
-                ({tokenData.symbol})
-              </Label>
-            </RowFixed>
-          </HideLarge>
-        </TokenText>
+        <ExtLink title={tokenData.name} href={getBlockExplorerUrl(chainId, tokenData.address, 'token')}>
```

Comment:
`title` is redundant with data displayed.
I suggest to either:
1. Display something else on hover (Symbol maybe?)
2. Do not show include `title` at all